codecov.yml

```diff
@@ -81,16 +81,10 @@ component_management:
         - src/neptune/logging/**
         - src/neptune/metadata_containers/**
         - src/neptune/new/**
-        - src/netpune/types/**
```

Comment:
@Raalsky how did this even work? :laughing:

Reply:
Ohh. Good catch!